parsers/mainz.py

```diff
@@ -124,13 +124,10 @@ def parse_data(canteen, data):
     if 'speiseplan_bldngall_name' in v['class']:
       canteen_name = str(v.string).strip()
       
-    if 'speiseplancounter' in v['class']:
-      # Save the countername as category to list meals by counter
-      category = str(v.string).strip()
```

Comment:
I think you meant to remove the if above that one? 😅 Otherwise category will always be None.

Reply:
Oh no, F****! I'm so sorry, correction is on the way.